src/sql/parts/notebook/notebook.component.ts

```diff
@@ -538,4 +540,17 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._model.pushEditOperations(edits);
 		return true;
 	}
+
+	public async runCell(cell: ICellModel): Promise<boolean> {
+		// TODO should ensure this cell is part of this notebook's model
```

Comment:
not sure this TODO is still applicable, since we're doing findIndex on this._model.cells below